.github/CODEOWNERS

```
@@ -50,14 +50,22 @@
 /packages/aws/data_stream/s3_daily_storage @elastic/obs-ds-hosted-services
 /packages/aws/data_stream/s3_request @elastic/obs-ds-hosted-services
 /packages/aws/data_stream/s3_storage_lens @elastic/obs-infraobs-integrations
-/packages/aws/data_stream/s3access @elastic/obs-ds-hosted-services @elastic/security-service-integrations
+/packages/aws/data_stream/s3access @elastic/obs-ds-hosted-services
 /packages/aws/data_stream/securityhub_findings @elastic/security-service-integrations
 /packages/aws/data_stream/securityhub_insights @elastic/security-service-integrations
 /packages/aws/data_stream/sns @elastic/obs-infraobs-integrations
 /packages/aws/data_stream/sqs @elastic/obs-infraobs-integrations
 /packages/aws/data_stream/transitgateway @elastic/obs-infraobs-integrations
 /packages/aws/data_stream/usage @elastic/obs-infraobs-integrations
 /packages/aws/data_stream/vpn @elastic/obs-infraobs-integrations
+/packages/aws/data_stream/firewall_logs @elastic/security-service-integrations
+/packages/aws/data_stream/firewall_metrics @elastic/security-service-integrations
```

I think the metrics data streams belong to observability.

```suggestion
/packages/aws/data_stream/firewall_metrics @elastic/obs-infraobs-integrations
```